oas_docs/kibana.info.yaml

```diff
@@ -2,6 +2,8 @@ openapi: 3.0.3
 info:
   title: Kibana APIs
   description: |
+    NOTE: Access to internal {kib} API endpoints will be restricted in Kibana version 9.0. For more information, refer to https://github.com/elastic/kibana/issues/163654.
```

Comment:
@lcawl how do we get this change into the oas?

Reply:
Note that there are lots more changes in the output files than just this PR caused, since we haven't got it running automatically for each PR yet.